packages/Operators/test/tstConsistentInterpolationOperator.cpp

```diff
@@ -302,7 +302,7 @@ TEUCHOS_UNIT_TEST( ConsistentInterpolationOperator, no_domain_0_test )
     for ( int i = 0; i < num_points; ++i )
     {
 	point_ids[i] = num_points*comm_rank + i;
-	point_dofs[i] = 0.0;
+	point_dofs[i] = 2.2;
```

Comment:
It was to make sure we would overwrite the data with zeros if we missed points. The test wasn't checking that before.

> On Apr 21, 2016, at 10:30 PM, dalg24 <notifications@github.com> wrote:
> 
> In packages/Operators/test/tstConsistentInterpolationOperator.cpp:
> 
> > @@ -302,7 +302,7 @@ TEUCHOS_UNIT_TEST( ConsistentInterpolationOperator, no_domain_0_test )
> >      for ( int i = 0; i < num_points; ++i )
> >      {
> >  	point_ids[i] = num_points*comm_rank + i;
> > -	point_dofs[i] = 0.0;
> > +	point_dofs[i] = 2.2;
> why did you change that one?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>